### PR TITLE
Fix multimodule errors introduced into .browser.js

### PR DESCRIPTION
**Pull request by cobbal**

### CODE REVIEW

File: Sources/FishyJoesNodeRuntime/js/__MODULE_NAME__.browser.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { WASI } from "@wasmer/wasi";
 import { WasmFs } from "@wasmer/wasmfs";
 import { NAPI } from "./wasm-napi.js";
 
-let __MODULE_EXPORTS__;
+let __MODULE_NAME__ __MODULE_DEPENDENCIES__;
```
Comment (kellyo-cricut) [inline Sources/FishyJoesNodeRuntime/js/__MODULE_NAME__.browser.js:10]:
```suggestion
let __MODULE_NAME__, __MODULE_DEPENDENCIES__;
```

File: Sources/FishyJoesNodeRuntime/js/__MODULE_NAME__.browser.js [hunk 2]
```diff
@@ -53,8 +53,8 @@ const init = async () => {
   console.log(instance);
   wasi.start(instance);
   const library = napi.init(instance);
-  { __MODULE_NAME__, __MODULE_DEPENDENCIES__} = library;
+  ({ __MODULE_NAME__ __MODULE_DEPENDENCIES__ } = library);
```
Comment (kellyo-cricut) [inline Sources/FishyJoesNodeRuntime/js/__MODULE_NAME__.browser.js:56]:
```suggestion
  ({ __MODULE_NAME__, __MODULE_DEPENDENCIES__ } = library);
```

File: Sources/FishyJoesNodeRuntime/js/__MODULE_NAME__.browser.js (hunk 1 above)
Comment (cobbal) [reply Sources/FishyJoesNodeRuntime/js/__MODULE_NAME__.browser.js:10]: __MODULE_DEPENDENCIES__ now puts a comma before each item instead of after

Comment (cobbal) [reply Sources/FishyJoesNodeRuntime/js/__MODULE_NAME__.browser.js:10]: because apparently `let a, b,;` is an error

**Review by kellyo-cricut — APPROVED** [review]

**Pull request merged**